src/vma/proto/dst_entry.h

```diff
@@ -120,6 +120,7 @@ class dst_entry : public cache_observer, public tostr, public neigh_observer
 	in_addr_t		m_route_src_ip; // source IP used to register in route manager
 	in_addr_t		m_pkt_src_ip; // source IP address copied into IP header
 	lock_mutex_recursive 	m_slow_path_lock;
+	lock_mutex_recursive 	m_tx_migration_lock;
```

Comment:
Do you see any reason to use recursive lock? Please consider using usual lock.

Reply:
Updated